capstone/src/main/java/com/google/univiz/api/representation/Deadline.java

```diff
@@ -1,10 +1,45 @@
 package com.google.univiz.api.representation;
 
 import com.google.auto.value.AutoValue;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
 
 /**
  * Contains the date when applications are due for a particular college. Will be made serializable
  * for the purpose of graphing due dates.
  */
 @AutoValue
-public abstract class Deadline {}
+@GenerateTypeAdapter
+public abstract class Deadline {
```

Comment:
Should there be a collegeId to tie it in to a particular college?

Reply:
The visualization will look something like this:
![Heat Map](https://user-images.githubusercontent.com/43079005/89064698-531bd000-d31f-11ea-9af3-6f7dbe73f359.png)

It will show how frequently and around what times applications are open/due, without attaching it to particular universities. That's why I didn't put the id or the name.

Reply:
SGTM